packages/core-components/src/components/paragraph/readme.md

```diff
@@ -12,11 +12,13 @@ Initial story: https://otto-eg.atlassian.net/browse/B2BDS-45
 
 ## Properties
 
-| Property | Attribute | Description                | Type                            | Default    |
-| -------- | --------- | -------------------------- | ------------------------------- | ---------- |
-| `align`  | `align`   | The alignment of the text. | `"center" \| "left" \| "right"` | `'left'`   |
-| `size`   | `size`    | The size of the text.      | `"100" \| "50"`                 | `'100'`    |
-| `weight` | `weight`  | The weight of the text.    | `"bold" \| "normal"`            | `'normal'` |
+| Property  | Attribute | Description                                                         | Type                            | Default    |
+| --------- | --------- | ------------------------------------------------------------------- | ------------------------------- | ---------- |
+| `align`   | `align`   | The alignment of the text.                                          | `"center" \| "left" \| "right"` | `'left'`   |
+| `margin`  | `margin`  | Whether or not the paragraph has a bottom margin. Defaults to true. | `boolean`                       | `true`     |
+| `size`    | `size`    | The size of the text.                                               | `"100" \| "50"`                 | `'100'`    |
+| `variant` | `variant` | The color of the paragraph. Defaults to black.                      | `"black" \| "grey"`             | `'black'`  |
```

Comment:
here the type can be black-100 and grey-400

Reply:
It's an autogenerated file, I'll rebuild and push the changes